lms/djangoapps/instructor/views/api.py

```diff
@@ -2629,8 +2636,20 @@ def spoc_gradebook(request, course_id):
         for student in enrolled_students
     ]
```

Comment:
Is this getting the grades for all students, even though we're only looking at the first page?